slotfills/sections/meta-box.jsx

```diff
@@ -5,58 +5,35 @@ import { PanelBody } from '@wordpress/components';
 import usePostMetaValue from '@/hooks/use-post-meta-value';
 
 // Components.
+import FieldRouter from './fieldRouter';
 import Group from './group';
-import TextField from './text-field';
-import TextareaField from './textarea-field';
 
 const MetaBox = ({
   field: {
     title,
     fm: context,
   },
-}) => {
-  let { children } = context;
-
-  if (Array.isArray(children)) {
-    children = [children];
-  }
-  const {
-    field_class: fieldClass,
-    attributes: {
-      rows = null,
-    } = {},
-    label = '',
-  } = context;
-  return (
-    <PluginDocumentSettingPanel
-      name={context.name}
-      title={title}
-    >
-      <PanelBody>
-        {fieldClass === 'text' && rows === null ? (
-          <TextField
-            field={context}
-            valueHook={usePostMetaValue}
-            label={label}
-          />
-        ) : null}
-        {fieldClass === 'text' && rows !== null ? (
-          <TextareaField
-            field={context}
-            valueHook={usePostMetaValue}
-            label={label}
-          />
-        ) : null}
-        {fieldClass === 'group' ? (
-          <Group
-            field={context}
-            valueHook={usePostMetaValue}
-          />
-        ) : null}
-      </PanelBody>
-    </PluginDocumentSettingPanel>
-  );
-};
+}) => (
+  <PluginDocumentSettingPanel
+    name={context.name}
+    title={title}
+  >
+    <PanelBody>
+      {context.field_class === 'group' ? (
+        <Group
+          field={context}
+          valueHook={usePostMetaValue}
+        />
+      ) : null}
+      {context.field_class !== 'group' ? (
+        <FieldRouter
+          field={context}
+          valueHook={usePostMetaValue}
+        />
+      ) : null}
```

Comment:
These ternaries can be collapsed since they're checking for opposite cases:
```suggestion
      {context.field_class === 'group' ? (
        <Group
          field={context}
          valueHook={usePostMetaValue}
        />
      ) : (
        <FieldRouter
          field={context}
          valueHook={usePostMetaValue}
        />
      )}
```